simplified-ui-accounts/src/main/java/org/nypl/simplified/ui/accounts/AccountDetailViewModel.kt

```diff
@@ -69,14 +71,20 @@ class AccountDetailViewModel(
   val buildConfig =
     services.requireService(BuildConfigurationServiceType::class.java)
 
-  val eula: EULAType? =
-    this.documents.eula
-
   val account =
     this.profilesController
       .profileCurrent()
       .account(this.accountId)
 
+  val eula: SimpleDocument? =
```

Comment:
Is there any reason this is declared as `SimpleDocument` instead of `DocumentType`?